occupancy-quickstart/src/actions/userDefinedFunctions/multiplemotionsensors.js

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// Sample code for detecting motion in an area with multiple motion sensors.
+
+var motionType = "Motion";
+var spaceAvailFresh = "AvailableAndFresh";
```

Comment:
If the value is indicating OccupancyStatus of the space, prefer renaming from AvailableAndFresh -> OccupancyStatus

Reply:
Will create a github issue as discussed in Teams.